testsuite/openmodelica/fmi/ModelExchange/2.0/fmi_attributes_11.mos

```diff
@@ -0,0 +1,97 @@
+// name: fmi_attributes_11.mos
+// keywords: FMI 2.0 export
+// status: correct
+// teardown_command: rm -rf fmi_attributes_11.fmu fmi_attributes_11.log fmi_attributes_11.xml fmi_attributes_11_tmp.xml fmi_attributes_11.fmutmp/
+
+loadString("
+model fmi_attributes_11
+  input Real u;
+  Real x;
+  output Real y;
+  Real a, b;
+  parameter Real z=30;
+initial equation
+  x = 0.0;
+equation
+  a = 2.0*u;
+  b = 0.5*a;
+  der(x) = b+z;
+  y = x;
+end fmi_attributes_11;
+"); getErrorString();
+
+setCommandLineOptions("--fmiFilter=blackBox"); getErrorString();
+translateModelFMU(fmi_attributes_11); getErrorString();
+
+// unzip to console, quiet, extra quiet
+system("unzip -cqq fmi_attributes_11.fmu modelDescription.xml > fmi_attributes_11_tmp.xml"); getErrorString();
+system("sed -n \"/<ModelVariables>/,/<\\/ModelVariables>/p\" fmi_attributes_11_tmp.xml > fmi_attributes_11.xml"); getErrorString();
+readFile("fmi_attributes_11.xml"); getErrorString();
+
+system("sed -n \"/<ModelStructure>/,/<\\/ModelStructure>/p\" fmi_attributes_11_tmp.xml > fmi_attributes_11.xml"); getErrorString();
+readFile("fmi_attributes_11.xml"); getErrorString();
+
+
+// Result:
+// true
+// ""
+// true
+// ""
+// "fmi_attributes_11.fmu"
+// ""
+// 0
+// ""
+// 0
+// ""
+// "  <ModelVariables>
+//   <!-- Index of variable = \"1\" -->
+//   <ScalarVariable
+//     name=\"x\"
```

Comment:
All names (except for inputs/outputs) should be concealed.

Reply:
How would you do that? Just create unique cryptic names?

Reply:
In Dymola they are using some thing like , concealed1, concealed2, just a random unique name

Reply:
this is implemented now see https://github.com/OpenModelica/OpenModelica/pull/880/commits/e932328bea48c761af84cb37ee2a30bb5afde67f